plasma_framework/contracts/src/framework/registries/ExitGameRegistry.sol

```diff
@@ -7,10 +7,10 @@ import "../utils/Quarantine.sol";
 contract ExitGameRegistry is Operated {
     using Quarantine for Quarantine.Data;
 
-    mapping(uint256 => address) private _exitGames;
-    mapping(address => uint256) private _exitGameToTxType;
-    mapping(uint256 => uint8) private _protocols;
-    Quarantine.Data private _exitGameQuarantine;
+    mapping(uint256 => address) public exitGames;
```

Comment:
ok, I think you convinced me. I prefer to use public vars since we should not use security through obscurity, but since we do a lot with inheritance it is ok I guess, provided that the only reason for using `private` field instead od a `public` one is to hide the field from the inheriting contracts ;)